core/base/ftmTree/FTMTreeUtils_Template.h

```diff
@@ -10,44 +10,35 @@
 
 #pragma once
 
-//#include <FTMTreeMT.h>
-
 namespace ttk {
   namespace ftm {
     // --------------------
-    // Get
+    // Utils
     // --------------------
+    template <typename type>
+    static type myAbs(const type var) {
+      return (var >= 0) ? var : -var;
+    }
 
     template <class dataType>
-    std::tuple<dataType, dataType> FTMTree_MT::getBirthDeath(idNode nodeId) {
-      idNode originId = this->getNode(nodeId)->getOrigin();
-      if(this->isNodeOriginDefined(
-           nodeId)) { // Avoid error if origin is not defined
-        dataType pers1 = this->getValue<dataType>(nodeId);
-        dataType pers2 = this->getValue<dataType>(originId);
-        dataType birth = std::min(pers1, pers2);
-        dataType death = std::max(pers1, pers2);
-        return std::make_tuple(birth, death);
-      }
-      return std::make_tuple(0.0, 0.0);
+    bool isEqual(dataType first, dataType two, double eps = 1e-6) {
```

Comment:
same comment here, isn't there something more standard (for such a simple test)?

Reply:
If I remember well I made this function because the comparison of double or float can be tricky some times. A simple `==` could not detect the equality of two double due to numerical precision, because they will be slighlty different at a very far decimal.

Reply:
Removed in the next commit.

Reply:
in next commit